db/rdbms/src/main/java/io/camunda/db/rdbms/read/domain/VariableDbFilter.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms.read.domain;
+
+import io.camunda.search.filter.VariableFilter;
+import io.camunda.search.filter.VariableValueFilter;
+import java.util.List;
+
+/**
+ * Since the operator fields in the VariableValueFilter are reserved keywords in mybatis (OGNL
```

Comment:
❓ I understand that operators like `eq,` `neq,` etc. are reserved keywords, but I don't fully understand why it is necessary to wrap them into another filter object. Additionally, couldn't we do something similar - what is done with `VariableValueDbFilter` - in the layer above already? So instead of having class properties like `eq` (and others) in the `VariableValueFilter`, wrap them in a record/tuple that contains the name, operator, and the value. Would that allow us to avoid having that additional filter`?

Reply:
Sure, this would be much better in the layer above. :) I just don't wanted to changed existing behaviour since this also effects the ES/OS Layer. I can do this directly with this PR.

Reply:
This class is not longer needed by using the new Operation Strategy which is already in the VariableFilter included. 

private Map<String, List<Operation<Object>>> variableOperations;